### DISCUSSION TITLE
Fresh Install of LazyVim, checkhealth showing error.

### DISCUSSION THREAD

**Discussion by uldrim** [LazyVim/LazyVim#991] [Q&A]
Can someone help me debug why checkhealth is showing this error.
I am on ubuntu 22.04 LTS and neovim 0.9 (stable)
Thanks.
![Screenshot from 2023-06-27 17-27-33](https://github.com/LazyVim/LazyVim/assets/45782936/841be25c-66df-476d-9e07-4fb5603cd4d4)

**Comment by folke** [comment]
I don't have that error.
Invalid VIMRUNTIME means that something is up with your Neovim installation

**Comment by azdanov** [comment]
Maybe try a clean install:
```sh
rm -rf ~/.config/nvim
rm -rf ~/.local/share/nvim
rm -rf ~/.local/state/nvim
rm -rf ~/.cache/nvim
```
